docs/quickstarts/packet.md

```diff
@@ -0,0 +1,225 @@
+
+Run the following command to create the cluster:
+
+```console
+lokoctl cluster install
+```
+
+Once the command finishes, your Lokomotive cluster details are stored in the path you've specified
+under `asset_dir`.
+
+## Verification
+
+A successful installation results in the output:
+
+```console
+module.packet-mycluster.null_resource.bootkube-start: Still creating... [4m10s elapsed]
+module.packet-mycluster.null_resource.bootkube-start: Still creating... [4m20s elapsed]
+module.packet-mycluster.null_resource.bootkube-start: Creation complete after 4m25s [id=1122239320434737682]
+
+Apply complete! Resources: 74 added, 0 changed, 0 destroyed.
+
+Your configurations are stored in /home/imran/lokoctl-assets/mycluster
+
+Now checking health and readiness of the cluster nodes ...
+
+Node                                          Ready    Reason          Message
+
+mycluster-controller-0                        True     KubeletReady    kubelet is posting ready status
+mycluster-controller-1                        True     KubeletReady    kubelet is posting ready status
+mycluster-controller-2                        True     KubeletReady    kubelet is posting ready status
+mycluster-worker-pool-1-worker-0              True     KubeletReady    kubelet is posting ready status
+mycluster-worker-pool-1-worker-1              True     KubeletReady    kubelet is posting ready status
+mycluster-worker-pool-1-worker-2              True     KubeletReady    kubelet is posting ready status
+
+Success - cluster is healthy and nodes are ready!
+```
+
+Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+
+```console
+export KUBECONFIG=./lokomotive-assets/auth/kubeconfig
+kubectl get nodes
+```
+
+## Cleanup
+
+To destroy the Lokomotive cluster, execute the following command:
+
+```console
+lokoctl cluster destroy --confirm
+```
+
+You can safely delete the working directory created for this quickstart guide if you no longer
+require it.
+
+## Troubleshooting
+
+### Stuck at copy controller secrets
+
+If there is an execution error or no progress beyond the output provided below:
+
+```console
+...
+module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m30s elapsed)
+module.packet-mycluster.null_resource.copy-controller-secrets: Still creating... (8m40s elapsed)
+...
+```
+
+The error is because the `ssh_pubkeys` provided in the configuration is missing in the `ssh-agent`
+
+To rectify the error, you need to:
+
+1. Execute [Cleanup](#cleanup) steps.
+2. Follow the steps [to add the SSH key to the
+   ssh-agent.](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent)
+3. Retry from Step 4:
+
+### Packet provisioning failed
+
+For failed machine provisioning on Packet end, execute the steps mentioned in the
+[Cleanup](#cleanup) section and retry [Step 5.](#step-5-create-lokomotive-cluster)
```

Comment:
If provisioning fails, you don't need to destroy everything. You can just re-run `install` command and it will re-run Terraform.